### ISSUE TITLE
How to divide the data to validation and test set?

### ISSUE THREAD

**Issue by hadi-ghnd** [endernewton/tf-faster-rcnn#131]
I have replaced the pascal data with a custom data and ran faster-rcnn on it for 50000 iterations and here are the **AP results**:
```
im_detect: 1/71 9.840s 0.001s
im_detect: 2/71 5.388s 0.001s
im_detect: 3/71 3.902s 0.000s
im_detect: 4/71 3.159s 0.000s
.
.
.
im_detect: 68/71 1.064s 0.000s
im_detect: 69/71 1.062s 0.000s
im_detect: 70/71 1.061s 0.000s
im_detect: 71/71 1.059s 0.000s
Evaluating detections
Writing car VOC results file
Writing pickup VOC results file
Writing tractor VOC results file
Writing boat VOC results file
Writing truck VOC results file
Writing campingcar VOC results file
Writing van VOC results file
Writing other VOC results file
Writing plane VOC results file
VOC07 metric? Yes
Reading annotation for 1/71
Saving cached annotations to /home/hadi/Software/tensorflow/tf-faster-rcnn/data/VOCdevkit2007/annotations_cache/annots.pkl
AP for car = 0.6870
AP for pickup = 0.8585
AP for tractor = 0.7092
AP for boat = 0.5455
AP for truck = 0.6364
AP for campingcar = 0.7851
AP for van = 0.6364
AP for other = 0.4394
AP for plane = 1.0000
Mean AP = 0.6997
```

The dataset has 1271 images. I set it this way:

- 800 images for training
- 400 images for validation
- 71 images for testing

But it seems like the **AP values** are computed based on the test images instead of validation set. What would be the best way to divide these images for validation and test?

**Comment by endernewton** [comment]
not sure what you mean, you can just set whichever set you want to test on, but kudos on running the pipeline successfully!
